src/main/java/ch/ipt/see/playground/MathController.java

```diff
@@ -0,0 +1,30 @@
+package ch.ipt.see.playground;
+
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class MathController {
+
+    @PostMapping(
+            path = "/math/volume-computation/hyperrectangle",
+            consumes = MediaType.APPLICATION_XML_VALUE,
+            produces = MediaType.APPLICATION_XML_VALUE
+    )
+    public ResponseEntity<CalculationResult> calculateHyperrectangleVolume(@RequestBody Calculation calculation) {
```

Comment:
gemäss https://ipt-training.atlassian.net/browse/SEE-3 müsste der Endpoint `/math/volume-computation/hyperrectangle` heissen
```suggestion
            path = "/math/volume-computation/hyperrectangle",
```